doc/nvim-tree-lua.txt

```diff
@@ -548,6 +549,10 @@ Show LSP and COC diagnostics in the signcolumn
     Icons for diagnostic severity.
       Type: `table`, Default: `{ hint = "", info = "", warning = "", error = "" }`
 
+    *nvim-tree.diagnostics.severity*
+    Controls the severity of the diagnostics.
```

Comment:
Controls it how? Feels like description lack detail. Do you mind expanding it a bit along lines 'set up minimum severity for which diagnostics will be displayed.'

Reply:
Thanks! I updated the description

Reply:
Thanks!